### PR TITLE
Add Analytics

### PR DESCRIPTION
**Pull request by rodericj**

### CODE REVIEW

File: SF iOS/SF iOS/Helpers/Analytics.m [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    CKDatabase *database = [[CKContainer defaultContainer] publicCloudDatabase];
+    CKRecord *record = [[CKRecord alloc] initWithRecordType:@"backgroundFetchComplete"];
+    [record setValuesForKeysWithDictionary:properties];
+    [database saveRecord:record
+       completionHandler:^(CKRecord * _Nullable record,
+                           NSError * _Nullable error) {
+           if (onCompletion) {
+               onCompletion(error);
+           }
+    }];
```
Comment (rodericj) [inline SF iOS/SF iOS/Helpers/Analytics.m:27]: This is the main point of this PR. We're capturing the state of the background fetch. Error, new data, no data. We write it to the `backgroundFetchComplete` record type. I'll figure out querying later.

**Review by mcritz — APPROVED** [review]

**Comment by rodericj** [comment]
Thanks @mcritz. I owe you.

File: SF iOS/SF iOS.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1078,10 +1090,11 @@
 			baseConfigurationReference = 95933610458B233CD7F79875 /* Pods-Coffup.debug.xcconfig */;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CODE_SIGN_ENTITLEMENTS = Coffup.entitlements;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 17;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = F4WTN36U2F;
```
Comment (rodericj) [inline SF iOS/SF iOS.xcodeproj/project.pbxproj:1097]: Dammit.

Comment (rodericj) [reply SF iOS/SF iOS.xcodeproj/project.pbxproj:1097]: This may be for CloudKit stuff actually. We may be in good shape

**Comment by rodericj** [comment]
Just so I can track this somewhere. I've deployed the app to test flight for some users and I've deployed the schema to production in CloudKit.

**Pull request merged**